uf3/regression/least_squares.py

```diff
@@ -684,6 +765,9 @@ def dataframe_to_tuples(df_features,
         y (np.ndarray): target vector.
         w (np.ndarray): weight vector for machine learning.
     """
+    if len(df_features) <= 1:
```

Comment:
Should be reverted (deleted). A past commit that allowed for singlepoint training got rid of this section and placed this check inside `calc_E_F_weights()`.